resources/assets/lib/components/comment.coffee

```diff
@@ -90,6 +97,10 @@ export class Comment extends React.PureComponent
       parent = store.comments.get(@props.comment.parentId)
       user = @userFor(@props.comment)
       meta = commentableMetaStore.get(@props.comment.commentableType, @props.comment.commentableId)
+      @isBlocked = isBlocked(user)
+
+      if !@props.comment.isDeleted && @isBlocked && !@state.forceShow
+        return @renderBlocked()
```

Comment:
The entire comment should be dimmed if the user is blocked, not just the placeholder.